### PR TITLE
Switch to `phylum init` for project setup

### PR DESCRIPTION
**Pull request by cd-work**
Since `phylum init` provides a better user experience than manually
creating projects yourself, we should recommend it over the other
alternatives.

### CODE REVIEW

File: docs/welcome/welcome.md [hunk 1]
```diff
@@ -21,49 +22,73 @@ Not a user yet? Get the free version [here](https://app.phylum.io/register).
...
 
 Resource: https://blog.phylum.io/insights-and-resources/pick-a-python-lockfile-and-improve-security
+
 ```sh
-phylum analyze <lockfile>
+phylum analyze
 ```
-![Analyze Project](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/submit-job.webp)
+
+![Analyze Project](TODO)
```
Comment (cd-work) [inline docs/welcome/welcome.md:69]: This video needs to be re-recorded without lockfile parameter.

File: docs/welcome/welcome.md [hunk 2]
```diff
@@ -21,49 +22,73 @@ Not a user yet? Get the free version [here](https://app.phylum.io/register).
...
+All analysis jobs *_must_* be associated with a Phylum project. To setup your
+project, run the following:
+
 ```sh
-phylum project create <projectName>
+phylum init
 ```
-![Create Project](https://raw.githubusercontent.com/phylum-dev/documentation/main/assets/create-project.webp)
+
+![Setup Project](TODO)
```
Comment (cd-work) [inline docs/welcome/welcome.md:56]: Need to record a video for `phylum init` here.

**Review by maxrake — DISMISSED** [review]
Approving, assuming the videos will be (re-)recorded and updated here first. Who is expected to do that? @cd-work, @furi0us333, or @louislang ?

**Comment by louislang** [comment]
> Approving, assuming the videos will be (re-)recorded and updated here first. Who is expected to do that? @cd-work, @furi0us333, or @louislang ?

Chatted with @cd-work, i've got that on my todo list for this evening. Will push the updated videos to this PR.

File: docs/welcome/welcome.md [hunk 3]
```diff
@@ -21,49 +22,73 @@ Not a user yet? Get the free version [here](https://app.phylum.io/register).
...
 
 ### Analyze your Project
-To begin analyzing your project for software supply chain risks, submit a lockfile to Phylum.
+
+To begin analyzing your project for software supply chain risks, submit your
+lockfile to Phylum.
```
[the remark this answers is not in the document]
Comment (cd-work) [reply docs/welcome/welcome.md:61]: I feel like using parenthesis just inhibits reading unnecessarily. If we wanted to imply multiple are fine too, we could just go with `lockfiles`.

**Review by maxrake — APPROVED** [review]
LGTM.

**Pull request merged**